kibana.json

```diff
@@ -1,7 +1,7 @@
 {
   "id": "opendistroSecurity",
   "version": "0.0.1",
-  "kibanaVersion": "8.0.0",
+  "kibanaVersion": "kibana",
```

Comment:
Is this always pointing to 7.9?

Reply:
this means always using current kibana version

Reply:
What does the "current" mean? Latest release? Is it 7.9 right now? Will it become 8 in the future?

Reply:
yeah, `kibana` means using the kibana version which plugin is built/running with, so we don't need to update this `kibana.json` file in future version upgrade